charts/microservice-chart/Chart.yaml

```
@@ -2,5 +2,5 @@ apiVersion: v2
 name: microservice-chart
 description: A Helm chart for PagoPA microservice
 type: application
-version: 1.20.3
+version: 1.21.0
```

```suggestion
version: 1.20.3
```
now is self managed with github ations